README.md

```diff
@@ -54,11 +54,11 @@ Alternatively, to install Mapbox Navigation using [Carthage](https://github.com/
 
 1. Create a [Cartfile](https://github.com/Carthage/Carthage/blob/master/Documentation/Artifacts.md#github-repositories) with the following dependency:
    ```cartfile
-   github "mapbox/mapbox-navigation-ios" ~> 0.29
+   github "mapbox/mapbox-navigation-ios" ~> 0.30
```

Comment:
custom-navigation.md also has similar code that needs updating.

Reply:
Never mind, it’s already been updated for v0.30.0.

Reply:
😎